bin/check_archived_drafts.py

```diff
@@ -150,6 +156,10 @@ def main():
     job_log(start, end, temp_dir, os.path.basename(__file__), messages=[message], status='Success')
     custom_print(f'end of {os.path.basename(__file__)} job')
 
+    # Cleaning created directory
+    if remove_all_yang_path:
+        os.rmdir(all_yang_path)
```

Comment:
Again, you can use `shutil.rmtree(all_yang_path, ignore_errors=True)` so `FileNotFoundError` is ignored.
Then you don't need to use `remove_all_yang_path` temp variable at all.